pycheribuild/projects/run_fvp.py

```diff
@@ -198,10 +198,8 @@ def execute_fvp(self, args: list, disk_image_path: Path = None, firmware_path: P
                 # and no way to recover otherwise anyway given we can't change
                 # port forwarding after running.
                 pre_cmd += ["-p", str(docker_host_ap_port) + ":" + str(default_ap_port)]
-            if ssh_port is not None:
-                pre_cmd += ["-p", str(ssh_port) + ":" + str(ssh_port)]
-            if gdb_port is not None:
-                pre_cmd += ["-p", str(gdb_port) + ":" + str(gdb_port)]
+            for p in tcp_ports:
+                pre_cmd += ["-p", p + ":" + p]
```

Comment:
Isn't tcp_ports actually a `typing.List[int]`? I think this needs to be
```suggestion
                pre_cmd += ["-p", str(p) + ":" + str(p)]
```